series-tvdb.en.yaml

```diff
@@ -1555,6 +1593,13 @@ entries:
       - season: 1
         anilist-id: 21499
 
+  - title: "Urusei Yatsura (2022)"
```

Comment:
Series is only called Urusei Yatsura on TVDB.

But you can add the (2022) one as synonym.